src/mlpack/methods/adaboost/loss_functions/square.hpp

```diff
@@ -0,0 +1,42 @@
+/**
+ * @file methods/adaboost/loss_functions/square.hpp
+ * @author Dinesh Kumar
+ * 
+ * Square loss class, which is a loss function for adaboost regressor.
+ *
+ * mlpack is free software; you may redistribute it and/or modify it under the
+ * terms of the 3-clause BSD license.  You should have received a copy of the
+ * 3-clause BSD license along with mlpack.  If not, see
+ * http://www.opensource.org/licenses/BSD-3-Clause for more information.
+ */
+
+#ifndef MLPACK_METHODS_ADABOOST_LOSS_FUNCTIONS_SQUARE_HPP
+#define MLPACK_METHODS_ADABOOST_LOSS_FUNCTIONS_SQUARE_HPP
+
+#include "mlpack.hpp"
+
+namespace mlpack {
+ 
+/**
+* The square loss is a loss function to measure the Beta value of the 
+* predictor and update the weights for the next machine in the ensemble
```

Comment:
I'm sure I've seen functions like this through the codebase.  Is there any way we can reuse some other class here?  Or come up with a unified approach?  Isn't this just MSE?  Or is there more to it that I've overlooked?